tests/src/Simple/Collision/Collision.il

```diff
@@ -0,0 +1,61 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+.assembly extern System.Runtime
```

Comment:
You should be able to write this test in C# using `/reference:[alias=]filename`. No need to resort to IL.